playbooks/roles/oraclejdk/defaults/main.yml

```diff
@@ -1,13 +1,15 @@
 ---
 
-oraclejdk_version: "8u65"
+oraclejdk_version: "8u131"
 # what the archive unpacks to
-oraclejdk_base: "jdk1.8.0_65"
-oraclejdk_build: "b17"
+oraclejdk_base: "jdk1.8.0_131"
+oraclejdk_build: "b11"
 oraclejdk_platform: "linux"
 oraclejdk_arch: "x64"
 oraclejdk_file: "jdk-{{ oraclejdk_version }}-{{ oraclejdk_platform }}-{{ oraclejdk_arch }}.tar.gz"
-oraclejdk_url: "http://download.oracle.com/otn-pub/java/jdk/{{ oraclejdk_version }}-{{ oraclejdk_build }}/{{ oraclejdk_file }}"
+
+oraclejdk_url: "http://download.oracle.com/otn-pub/java/jdk/{{ oraclejdk_version }}-{{ oraclejdk_build }}/d54c1d3a095b4ff2b6607d096fa80163/{{ oraclejdk_file }}"
```

Comment:
FYI: I'm concerned about the HEX value d54c1d3a095b4ff2b6607d096fa80163 . This won't be a problem in the short-term so i'm not blocking on this, but I want to explain why this may begin failing in the future. This is more of a knowledgeshare than a complaint

Some background: there are reports on github about these HEX values being uniquely generated for particular oracle accounts and suspended over time (therefore, using them will result in the authentication boundary). I don't think this will cause any issues for a long while because this particular JDK version won't be considered "archived" for a long while